### PR TITLE
add README.md

### PR DESCRIPTION
**Pull request by BohdanTrue**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+- [Docker](https://www.docker.com/): provides a consistent and reproducible way to deploy applications across different environments.
+- [Postman](https://www.postman.com/): allows developers to create and send HTTP requests to test APIs, monitor responses, and automate testing workflows.
+
+###🚦Controllers
+- _AuthenticationController_: handles user registration and authorization.
+- _BookController_: handles book-related tasks, including the creation, modification, deletion, and retrieval of book information.
+- _CartController_: oversees the status of the shopping cart, enabling users to add, modify, fetch, and remove items from the cart.
+- _CategoryController_: handles category management, permitting administrators to perform actions such as creation, modification, retrieval, and deletion.
+- _OrderController_: manages order-related tasks, including the creation, modification, deletion, and retrieval of order history.#Online BookStore API
+  ___A comprehensive Library Management API tailored for seamless integration into digital platforms, perfect for powering online book repositories and literary platforms📚.___
```
Comment (yevheniisynytsia) [inline README.md:26]: I guess something went wrong and you have duplicated text

File: README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+- [Swagger](https://swagger.io/): provides a framework for generating interactive API documentation, allowing developers to understand, test, and use APIs more easily.
+- [Docker](https://www.docker.com/): provides a consistent and reproducible way to deploy applications across different environments.
+- [Postman](https://www.postman.com/): allows developers to create and send HTTP requests to test APIs, monitor responses, and automate testing workflows.
+
+###🚦Controllers
+- _AuthenticationController_: handles user registration and authorization.
+- _BookController_: handles book-related tasks, including the creation, modification, deletion, and retrieval of book information.
+- _CartController_: oversees the status of the shopping cart, enabling users to add, modify, fetch, and remove items from the cart.
+- _CategoryController_: handles category management, permitting administrators to perform actions such as creation, modification, retrieval, and deletion.
+- _OrderController_: manages order-related tasks, including the creation, modification, deletion, and retrieval of order history.
```
Comment (yevheniisynytsia) [inline README.md]: what about sections:

* Endpoints and what role can access them with a short description.
* Do you have a postman collection? If so, include it in the readme.
* How to run and test your application.
* What challenges you've faced and how you overcome them.
* Possible improvements.

**Review by yevheniisynytsia — CHANGES_REQUESTED** [review]
Left some comments to work on =)

File: README.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
+#Online BookStore API
+___A comprehensive Library Management API tailored for seamless integration into digital platforms, perfect for powering online book repositories and literary platforms📚.___
+
+###👨‍💻Tech stack
```
Comment (d1sam) [inline README.md:4]: what about testing technologies?

File: README.md
Comment (d1sam) [inline README.md:1]: please add possible improvements section.

**Review by d1sam — CHANGES_REQUESTED** [review]
good job, but some changes still should be done!

File: README.md [hunk 4]
```diff
@@ -73,4 +76,8 @@ To overcome these challenges and establish a robust and secure authentication an
 
 By solving these challenges and implementing solutions, a secure and reliable authentication and authorization system using Spring Security was established. This ensures the protection of user data and sensitive operations within our Online Book Store API, creating a secure and trustworthy environment for users.
 
+###⚙️Possible Improvements
```
Comment (yevheniisynytsia) [inline README.md:79]: this is more "contribute to the project", but not "possible improvements"
you can add here smth like "I am going to make better search functionality, going to add a possibility to return the order" and so on

but you can also leave the text you have below the possible improvements, looks good!

**Review by yevheniisynytsia — APPROVED** [review]
Well done! 1 comment left, please consider working on it

**Pull request merged**